src/components/ReviewMigration.tsx

```diff
@@ -45,55 +50,67 @@ function ReviewMigration({
   const shouldDisplayGas = !!gasStandard
 
   return (
-    <div className="reviewMigration">
-      <h3>{t("reviewMigration")}</h3>
-      <div className="migrateTop">
-        <Warning>{t("migrationExplain")}</Warning>
-        <div className="info">
-          <div className="row">
-            <span className="title">{t("migrationAmount")}</span>
-            <span className="value floatRight">
+    <React.Fragment>
+      <DialogTitle variant="h1" onClose={onClose}>
+        {t("reviewMigration")}
+      </DialogTitle>
+      <DialogContent>
+        <Alert variant="filled" icon={false} severity="warning">
+          {t("migrationExplain")}
+        </Alert>
+        <Box my={3}>
+          <Box display="flex">
+            <Typography component="span">{t("migrationAmount")}</Typography>
```

Comment:
What is the reason for explicitly making passing `span` into this prop? Does it need it?

Reply:
Oh, the component props render `Typography` as that component. So the result will be `span`.
Because this component was `span` so I gave this props.

but as you said because it was wrapped with flex box, so that props don't mandatory.

Reply:
Are you saying by default Typography renders as a `span`?

Reply:
Agreed!